app/controllers/api/v2/allocations_controller.rb

```diff
@@ -7,15 +7,21 @@ class AllocationsController < API::V2::ApplicationController
       def index
         authorize Allocation
 
-        render jsonapi: policy_scope(Allocation.where(accredited_body_id: accredited_body.id)),
+        scope = Allocation.where(accredited_body_id: accredited_body.id)
+
+        if params[:filter] && params[:filter][:training_provider_code]
+          scope = scope.where(provider_code: params[:filter][:training_provider_code])
+        end
```

Comment:
I'm finding this a little confusing ...

In the PR guidance it states the following 
```
Allocation.where(recruitment_cycle_year: recruitment_cycle.year)
  .where(provider_code: provider.provider_code, training_provider_code: training_provider[:provider_code])
   .first
```

But in the if statement we are passing `provider_code: [training_provider_code]` ?

I may be missing something here ...

Reply:
no. because in publish the `provider_code` is the accredited body and not the training provider

Reply:
Ah right  I see now ... So confusing. I guess this publish / api inconsistency will addressed when we get around to this - https://trello.com/c/VCOC2FlP/36-make-accreditedprovider-etc-naming-consistent 🤞